packages/js-sync-with-portabl/src/lib/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface Options {
+  env: 'dev' | 'production';
+  envOverride?: {
```

Comment:
How do you feel about just allowing us to override without specifying local:

something just like:
```typescript
envOverride?: {
  domain: string;
  audience: string;
  passportUrl: string;
  syncAcceptUrl: string;
};
  ```
  
I'm also curious if we can strip this out at build time -- so that it's only available during development.

Reply:
As discussed let's not worry about stripping it out at build time for now.